pyproject.toml

```diff
@@ -38,7 +38,7 @@ arrow = "0.12.1"
 appdirs = "1.4.3"
 attrs = ">=19.3.0"
 bitstruct = "6.0.0"
-boto3 = "^1.26.77"
+boto3 = "1.17"
```

Comment:
The `boto3` dependency is being downgraded from 1.26.77 to 1.17.x. Is this intentional?

Reply:
So `boto3` dependency just needs to match the other repos to avoid `SolverProblemError` when poetry is resolving dependencies. I'll revert this back to `1.26.77` and do the same for other repos that use `epyqlib`
![image](https://github.com/epcpower/stlib/assets/91996247/96032896-8de7-4cb5-9596-b5b79e6e99c4)

Reply:
[e13da72](https://github.com/epcpower/stlib/pull/47/commits/e13da7243d47654e8029767e6085891197625e65)